package.json

```diff
@@ -11,6 +11,7 @@
   "main": "umd/skatejs.js",
   "module": "es/index.js",
   "es2015": "es-latest/index.js",
+  "esnext": "src/index.js",
```

Comment:
I think the main things I want are:

1. Consistency between the directory name and the package.json key.
2. Targeting the latest spec.

No point in having several versions of ES built.